.gitignore

```diff
@@ -1,4 +1,4 @@
-.DS_Store
+F.DS_Store
```

Comment:
is this intentional?

Reply:
fixed in a7d714b9ecf17d3c378053949179691971d1cc3f